source/RialtoGStreamerMSEAudioSink.cpp

```diff
@@ -340,7 +361,7 @@ static void rialto_mse_audio_sink_class_init(RialtoMSEAudioSinkClass *klass)
     gobjectClass->set_property = rialto_mse_audio_sink_set_property;
     elementClass->change_state = rialto_mse_audio_sink_change_state;
 
-    g_object_class_install_property(gobjectClass, PROP_VOLUME,
+    g_object_class_install_property(gobjectClass, PROP_VOLUME, PROP_MUTE,
```

Comment:
Does it even compile? :P You need separate g_object_class_install_property for mute property.